pages/common/picom-trans.md

```diff
@@ -0,0 +1,20 @@
+# picom-trans
+
+> Set window opacity.
+> More information: <https://github.com/yshui/picom>.
+
+- Set opacity of current window to 90 percent:
+
+`picom-trans --current --opacity={{90}}`
+
+- Set opacity of window with name 'Firefox' to 90 percent:
```

Comment:
```suggestion
- Set a specific window opacity:
```

Reply:
Something like this? :thinking: 
```suggestion
- Set an opacity for a window with a specific name:
```

Reply:
Yeah, sounds nicer than my variant.

Reply:
I'd use the definite article, however: `Set the opacity ...`